src/components/searchInput/searchInput.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { useEffect, useRef, useState } from 'react';
+import { Person } from '../../types/Person';
+
+interface Props {
+  delay: number;
+  setFullField: React.Dispatch<React.SetStateAction<string>>;
+  setSelectedPerson: React.Dispatch<React.SetStateAction<null | Person>>;
+  selectedPerson: null | Person;
+  setFocus: React.Dispatch<React.SetStateAction<boolean>>;
+  setDelay?: React.Dispatch<React.SetStateAction<number>>;
```

Comment:
The `setDelay` prop is defined as optional, but it is not used anywhere in the component. If you intend to allow users to change the delay dynamically, ensure that `setDelay` is utilized appropriately.